README.md

```diff
@@ -1,5 +1,7 @@
 # React Portal
 
+This library provides the ability to render children into a node that can exist outside the direct hierarchy of the parent component.
```

Comment:
Maybe just a short message to explain why the regular React Portal doesn't work for some use cases?

Reply:
@scotttrinh 7971485